_generator/types.yaml

```diff
@@ -393,7 +393,7 @@ types:
     anchor: service-order-note
   ordernotetype:
     file: serviceordernotes.md
-    anchor: service-order-note-type
+    anchor: ordernotetype
```

Comment:
question: Are you sure this anchor is correct? Actually, do we still need this entry?